packages/truffle-db/src/workspace/test/index.ts

```diff
@@ -1,6 +1,7 @@
 import path from "path";
 
-import { graphql } from "graphql";
+import gql from "graphql-tag";
```

Comment:
So if I understand right, the benefit of using `graphql-tag` is that it will parse our query strings into the standard graphQL AST? We were already using template literals, right? So now we just prefix with `gql`?